newIDE/app/src/InstancesEditor/HighlightedInstance.js

```diff
@@ -47,5 +56,37 @@ export default class InstancesSelection {
       highlightRectangle.height
     );
     this.highlightRectangle.endFill();
+
+    const tooltipInfo =
+      'Object: ' +
+      this.highlightedInstance.getObjectName() +
+      '\n' +
+      'X: ' +
+      parseInt(this.highlightedInstance.getX()) +
+      '  Y: ' +
+      parseInt(this.highlightedInstance.getY()) +
+      '\n' +
+      'Layer: ' +
+      this.highlightedInstance.getLayer() +
+      '  Z: ' +
+      this.highlightedInstance.getZOrder() +
+      '\n';
+    this.tooltip.text = tooltipInfo;
+    this.tooltip.x = //alternatively we could draw it at mouse position instead. Would help for big objects
+      highlightRectangle.x -
+      this.tooltip.width / 2 +
+      highlightRectangle.width / 2;
+    this.tooltip.y = highlightRectangle.y - 68;
+
+    this.tooltipBG.clear();
+    this.tooltipBG.beginFill(0x000000, 0.8);
+    this.tooltipBG.drawRoundedRect(
+      this.tooltip.x - 5,
```

Comment:
Use a constant instead of magic numbers:
`const padding = 5`;

replace 5 by `padding`
replace 10 by `2*padding`

(more generally, in code it's better to name constants instead of spreading them in the code as numbers: https://refactoring.guru/replace-magic-number-with-symbolic-constant)

Reply:
yes, you're right :)